### DISCUSSION TITLE
External Module's tables created in new Sites

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#1595] [Q&A]
Hi,

I've created a Custom Module project with dbcontext of Country model and deployed to Oqtane. Deployed custom module added in a page in Default site (1st site). Then I've created new Site2 from Site Management and it has new MSSQL database. I've looked into the Site2 database and it contains Country table. Actually it should not exists in Site2 database right?

**Comment by sbwalker** [comment]
This is by design. Modules are available to all Sites within an installation - regardless of whether the Sites are part of a shared database or are using isolated databases.

**Reply by Sharafudeen** [reply]
If the modules added to the site (in my case Site2), then only respective tables of module can create in the Site2 database will be good I think. To do this, where do I've to modify in the Oqtane framework? Is this good way to do?
